tests/opwen_email_server/integration/test_celery.py

```diff
@@ -38,3 +40,13 @@ def test_send_message(self):
                 routing_key=self.routing_key,
                 declare=[self.queue],
             )
+
+    def test_celery_task_routes(self):
+        registered_celerey_tasks = [
+            task_name for task_name in current_app.tasks.keys() if not task_name.startswith('celery.')
+        ]
+
+        for task_name in registered_celerey_tasks:
+            self.assertIs(
+                task_name in app.task_routes.keys(), True,
```

Comment:
Note that `key in dictionary.keys()` is equivalent to `key in dictionary` but the latter reads more naturally.